service-commands/scripts/setup-instance.sh

```diff
@@ -107,8 +107,29 @@ case "$service" in
 
 		read -p "Configure local /etc/hosts? [y/N] " -n 1 -r && echo
 		if [[ "$REPLY" =~ ^[Yy]$ ]]; then
-			sudo -E AUDIUS_REMOTE_DEV_HOST=$(get_ip_addr $provider $name) node $PROTOCOL_DIR/service-commands/scripts/hosts.js
-			exit 1
+			IP=$(get_ip_addr $provider $name)
+			echo "export AUDIUS_REMOTE_DEV_HOST=${IP}" >> ~/.zshenv
+			sudo -E AUDIUS_REMOTE_DEV_HOST=${IP} node $PROTOCOL_DIR/service-commands/scripts/hosts.js add-remote-host
+		fi
+
+		read -p "Set SSH ServerAliveInterval to 60? [y/N] " -n 1 -r && echo
+		if [[ "$REPLY" =~ ^[Yy]$ ]]; then
+			echo "ServerAliveInterval 60" | sudo tee -a /etc/ssh/ssh_config.d/60-audius.conf
+		fi
+
+		read -p "Setup zsh? [y/N] " -n 1 -r && echo
+		if [[ "$REPLY" =~ ^[Yy]$ ]]; then
+			execute_with_ssh $provider $user $name "sudo chsh -s /bin/zsh \"$USER\""
```

Comment:
I think the `$user` here needs to be $user or `\$USER` since we need the remote user, or we could use single quotes here?